src/components/Auth/LoginForm.js

```diff
@@ -0,0 +1,118 @@
+import React, { useState, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Link, Redirect } from 'react-router-dom';
+import { Box, Button, TextField } from '@material-ui/core/';
+import { useAuth } from './AuthContext';
+import axios from 'axios';
+
+const LoginForm = ({ toggleActiveForm }) => {
+  const [username, setUsername] = useState('');
```

Comment:
Would it be more explicit to set state as `nil` instead of an empty string?

Reply:
(Done)